pkg/api/customization/cluster/actionhandler.go

```diff
@@ -54,11 +54,18 @@ func (a ActionHandler) ClusterActionHandler(actionName string, action *types.Act
 		return a.ImportYamlHandler(actionName, action, apiContext)
 	case "exportYaml":
 		return a.ExportYamlHandler(actionName, action, apiContext)
+	}
+
+	if err := apiContext.AccessControl.CanDo(v3.ClusterGroupVersionKind.Group, v3.ClusterResource.Name, "update", apiContext, nil, apiContext.Schema); err != nil {
```

Comment:
Try not to return error here. I think It would be better to return `can-not-access` error in 
```
case "enableMonitoring":
  // test CanDo
  if err!=nil {
    return err
  }
  return a.enableMonitoring(actionName, action, apiContext)
```

Reply:
The result is the same but it would be nicer to other guys who try to change the logic here.